automated install/basic-install.sh

```diff
@@ -1287,8 +1287,7 @@ chooseBlocklists() {
     # Let user select (or not) blocklists via a checklist
     cmd=(whiptail --separate-output --checklist "Pi-hole relies on third party lists in order to block ads.\\n\\nYou can use the suggestions below, and/or add your own after installation\\n\\nTo deselect any list, use the arrow keys and spacebar" "${r}" "${c}" 5)
     # In an array, show the options available (all off by default):
-    options=(StevenBlack "StevenBlack's Unified Hosts List" on
-        MalwareDom "MalwareDomains" on)
+    options=(StevenBlack "StevenBlack's Unified Hosts List" on)
```

Comment:
Do we really need the whiptail select if there's only one choice?

Reply:
It still gives a new user /reconfigurerer a chance to de-select the list, I guess

Reply:
I second this.

Reply:
Should make it a radio list then, if there's only one binary choice.

Reply:
Have tested a `radiolist`, and it does not allow one to deselect if it is a single item. `checklist` is sufficient, I think.

@yubiuser cam you also update line 1288 to read:

```bash
cmd=(whiptail --separate-output --checklist "Pi-hole relies on third party lists in order to block ads.\\n\\nYou can use the suggestion below, and/or add your own after installation\\n\\nTo deselect the suggested list, use spacebar" "${r}" "${c}" 5)
```

Reply:
Hrm, I thought radio was yes/no.

Reply:
It seems that it is "Here are several options, you must choose one of them", with options only having an `off` state if one of the other options is in the `on` state. If there are no other options, then there is nothing to toggle it as `off`. Which makes some kind of sense.

Reply:
We could change it to radio and offer 
1) StevenBlack
2) No Adlist

Reply:
Yeah, I was going to suggest that... but that would need to be done in a separate PR - we would need to redo some of the code further on once a selection is made.

Reply:
How likely is it that there will be at least two adlists in the future... just asking before the code is changed here and later reverted back to the current selection menu.

Reply:
That, indeed, was another hesitation I had about making the suggestion ;)

Never say never, but probably never.

Reply:
I've emailed the Contact Me to ask about the replacement list.

Reply:
Usually we recommend to look at `https://firebog.net/` if users ask for lists. Why don't include we some of them already? There have been ~5 lists previously - based on what have they been chosen? Can't the same requirements be used to add some of the fireborg lists?